ocamlformat.opam

```
@@ -8,7 +8,7 @@ authors: ["Josh Berdine <jjb@fb.com>"]
 homepage: "https://github.com/ocaml-ppx/ocamlformat"
 bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
 depends: [
-  "ocaml" {>= "4.08" & < "4.15"}
+  "ocaml" {>= "5.00"}
```

```suggestion
  "ocaml" {>= "4.08"}
```
Compatible with 4.08, no needs to restraint the range.